org.gimp.GIMP.json

```diff
@@ -738,6 +741,8 @@
                 {
                     "name": "libde265",
                     "config-opts": [
+                        "--disable-dec265",
+                        "--disable-encoder",
```

Comment:
So we don't need these because we enable other decoder/encoder which work better and which we prefer over these, is that it?

Reply:
Those are extra tools so we can skip building them. We need just the library.